README.md

```diff
@@ -1,22 +1,33 @@
 # ChugSplash
 
-A declarative and deterministic framework for deploying and upgrading smart contracts. Available as a Foundry and Hardhat plugin.
+A declarative and deterministic framework for deploying and upgrading smart contracts. Available for both Hardhat and Foundry.
+
+Powers [ChugSplash Managed](https://www.chugsplash.io).
 
 > **WARNING**: The code and contracts ChugSplash uses to deploy and upgrade your contracts HAVE NOT been audited. ChugSplash is a BETA product undergoing significant active development. ChugSplash's behavior and APIs are subject to change at any time at our discretion. You should not use ChugSplash if you would be very upset with your project breaking without notice. We make no guarantees about the safety of any contract deployments using the ChugSplash system.
 
 If you want to use ChugSplash in production, ask a question, or request a feature then we'd love to hear from you in the [Discord!](https://discord.com/invite/CqUPhgRrxq)
 
-## Key Features
+## Features
 
-- Deploy new upgradeable contracts
-- Upgrade existing contracts
-- Fully deterministic and idempotent deployments
-- Declarative syntax for a better developer experience
-- Compatible with contracts deployed using the [OpenZeppelin Hardhat Upgrades API](https://docs.openzeppelin.com/upgrades-plugins/1.x/api-hardhat-upgrades)
+### ChugSplash Core Protocol
+- Define deployments and upgrades [declaratively](https://github.com/chugsplash/chugsplash/blob/develop/docs/chugsplash-file.md#layout-of-a-chugsplash-file) in a single file, inspired by [Terraform](https://www.terraform.io/).
+- Deployments are fully atomic, deterministic, and idempotent. Deployments cannot be halted for any reason and are finalized in a single transation.
 - Built-in storage layout safety checker
-- Automatically verifies contracts on Etherscan
-- Deploys contracts at the same addresses across networks via `CREATE2`
+- Deploys contracts at consistent addresses across networks using `CREATE2`
+- Keep track of contract dependencies using simple template syntax (i.e. `{{ MyContract }}`).
 - Generates deployment artifacts in the same format as hardhat-deploy
+- Compatible with contracts deployed using the [OpenZeppelin Hardhat Upgrades API](https://docs.openzeppelin.com/upgrades-plugins/1.x/api-hardhat-upgrades)
```

Comment:
Should we explicitly say OpenZeppelin Transparent and UUPS proxies? Hackathon users won't care but people who actually want to import their contracts will want to know